draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -45,15 +46,18 @@ informative:
 
 --- abstract
 
-This document defines a new verifiable data structure type for COSE Signed Merkle Tree Proofs specifically designed for implementations that rely on Trusted Execution Environments (TEEs) to provide stronger tamper-evidence guarantees.
+This document defines a new verifiable data structure type for COSE Signed Merkle Tree Proofs specifically designed for transaction ledgers produced by Trusted Execution Environments (TEEs) to provide stronger tamper-evidence guarantees.
 
 --- middle
 
 # Introduction
 
-The Concise Encoding of Signed Merkle Tree Proofs (CoMeTre) {{-COMTRE}} defines a common framework for defining different types of proofs, such as proof of inclusion, about verifiable data structures (also abbreviated as "logs" in this document). For instance, inclusion proofs guarantee to a verifier that a given serializable element is recorded at a given state of the log, while consistency proofs are used to establish that an inclusion proof is still consistent with the new state of the log at a later time.
+The Concise Encoding of Signed Merkle Tree Proofs (CoMeTre) {{-COMTRE}} defines a common framework for defining different types of proofs, such as proof of inclusion, about verifiable data structures (VDS). For instance, inclusion proofs guarantee to a verifier that a given serializable element is recorded at a given state of the VDS, while consistency proofs are used to establish that an inclusion proof is still consistent with the new state of the VDS at a later time.
```

Comment:
@OR13 I saw in #3 that you suggested adding references, which seems like an excellent idea. Do you have some pointers on the best way to add references in this kind of document? Just a link to the project's GitHub/paper?

Reply:
Here is an example of how to add an arbitrary informative reference:

https://github.com/cose-wg/draft-ietf-cose-dilithium/pull/5/files#diff-2478aa1dfa824cbb11926553a0f4f8c9cd3e223126798f02e4827a8a56e809e9R62